Makefile

```diff
@@ -40,24 +40,25 @@ default: help
 help:
 	@echo "Usage: make <target>"
 	@echo
-	@echo " * 'install'          - Install binaries to system locations"
-	@echo " * 'binaries'         - Build cri-containerd and ctrcri"
-	@echo " * 'plugin'           - Build cri-containerd as a plugin package"
-	@echo " * 'static-binaries   - Build static cri-containerd and ctrcri"
-	@echo " * 'release'          - Build release tarball"
-	@echo " * 'push'             - Push release tarball to GCS"
-	@echo " * 'test'             - Test cri-containerd with unit test"
-	@echo " * 'test-integration' - Test cri-containerd with integration test"
-	@echo " * 'test-cri'         - Test cri-containerd with cri validation test"
-	@echo " * 'test-e2e-node'    - Test cri-containerd with Kubernetes node e2e test"
-	@echo " * 'clean'            - Clean artifacts"
-	@echo " * 'verify'           - Execute the source code verification tools"
-	@echo " * 'proto'            - Update protobuf of cri-containerd api"
-	@echo " * 'install.tools'    - Install tools used by verify"
-	@echo " * 'install.deps'     - Install dependencies of cri-containerd (containerd, runc, cni) Note: BUILDTAGS defaults to 'seccomp apparmor' for runc build"
-	@echo " * 'uninstall'        - Remove installed binaries from system locations"
-	@echo " * 'version'          - Print current cri-containerd release version"
-	@echo " * 'update-vendor'    - Syncs containerd/vendor.conf -> vendor.conf and sorts vendor.conf"
+	@echo " * 'install'          		- Install binaries to system locations"
+	@echo " * 'binaries'         		- Build cri-containerd and ctrcri"
+	@echo " * 'static-binaries   		- Build static cri-containerd and ctrcri"
+	@echo " * 'release'          		- Build release tarball"
+	@echo " * 'push'             		- Push release tarball to GCS"
+	@echo " * 'test-containerd'  		- Build a customized containerd with CRI plugin for testing"
```

Comment:
/s/test-//

Reply:
I want to emphasize that this is not a real containerd, it's just for testing.

Reply:
Can you come up with a better name? My naming is bad.